src/main/java/Duke.java

```diff
@@ -35,17 +25,6 @@ public void run() throws InvalidCommandException {
     }
 
 
-    public void Exit() {
```

Comment:
Same for this method, should the method name be `exit()` instead of `Exit()`

Reply:
Thanks for pointing out!